application/application.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/wissance/Ferrum/config"
 	"github.com/wissance/Ferrum/data"
 	"github.com/wissance/Ferrum/logging"
-	"github.com/wissance/Ferrum/managers"
+	"github.com/wissance/Ferrum/managers/file_data_manager"
```

Comment:
thi is actually bad because we declare here on imports level that we work with file data manager only